src/main/kotlin/org/wycliffeassociates/otter/common/data/workbook/ResourceContainerInfo.kt

```diff
@@ -0,0 +1,6 @@
+package org.wycliffeassociates.otter.common.data.workbook
+
+data class ResourceContainerInfo(
```

Comment:
ResourceInfo is almost a better name here in that ResourceContainerMetadata is so similar, and because this doesn't exactly have anything to do with ResourceContainers (it is referring to the slug and title of a resource).

However, perhaps this highlights a problem we've overlooked in the past that applies to more things? Title and slug are largely separate fields for use of hash lookups, sometimes for apis but often to separate internal app use and user facing text (ie, already localized). As a result virtually everything has a slug and a title field.

Perhaps the data class here is just... Title; which exists in the data model package?

Not sure how I feel about that, just my initial thoughts.

Reply:
My hesitancy comes from the existence of this class:
https://github.com/WycliffeAssociates/otter-common/blob/dev/src/main/kotlin/org/wycliffeassociates/otter/common/data/model/ResourceMetadata.kt

Though admittedly I had thought it was named ResourceContainerMetadata.

Reply:
Renamed as `ResourceInfo`.